core/templates/dev/head/pages/practice-session-page/practice-session-page.module.ts

```diff
@@ -55,6 +56,11 @@ import { PracticeSessionPageConstants } from
     ServiceBootstrapComponent
   ],
   providers: [
+    {
```

Comment:
Why the interceptor is being installed here?

Reply:
@bansalnitish This is the module for the service that i am migrating, Finding the module for some services is tricky, I did some hacks to find the module and i will update the doc once I find that the method that I used is reliable. Having said that, it is working in this case.

Reply:
Could you still list how did you find this module? 

Might be useful in other services too.

Reply:
@bansalnitish Ok so there's something called as SkipSelf() decorator in Angular, which can be used while injecting the service in the constructor. It's more of a hack according to our use case, upon using that @Skipself() with the HttpClient dependency will give us an error along with the name of the module from which the service is being called.
![hack1](https://user-images.githubusercontent.com/30312043/74168887-68f1e780-4c50-11ea-935f-a150e684e36a.png)

Reply:
Aha, I see -- this is useful. Thanks @Showtim3 

cc/ @nishantwrp for  info.